### PR TITLE
Translation - Add Missing German

### PR DESCRIPTION
**Pull request by Nilia119**
Add Missing German Translation and improved it.

While I did that, I discovered something:
Unbenannt
According to the Stringtext (in addons/compat_cup_weapons/compat_cup_weapons_nightvision/stringtable.xml) the NVG is supposed to be Green, but almost in all translation it is titled as "Black" German included (except for Russian [I don't know about the Chinese/Korean/Japan one, i can't read that xD]).
I would change that accordingly.
Have a nice day!

### CODE REVIEW

File: addons/arsenal/stringtable.xml [hunk 1]
```diff
@@ -1253,6 +1254,7 @@
             <Russian>Интегрирован тепловизор и осн.прицел.</Russian>
             <Korean>열화상과 주무기 내장</Korean>
             <French>Thermique et primaire intégrés</French>
+            <German>Thermal &amp; Primär integriert</German>
```
Comment (BaerMitUmlaut) [inline addons/arsenal/stringtable.xml:1257]: "Primär integriert" versteht man denke ich nicht. Primary steht hier für die Hauptwaffe.

File: addons/common/stringtable.xml [hunk 2]
```diff
@@ -1834,20 +1834,23 @@
             <Korean>무기 흔들림</Korean>
             <French>Oscillation de l'arme</French>
             <Russian>Колебание оружия</Russian>
+            <German>Waffen Wackeln</German>
```
Comment (BaerMitUmlaut) [inline addons/common/stringtable.xml:1837]: Können wir hier eine bessere Übersetzung finden? Waffenunruhe vielleicht?

File: addons/common/stringtable.xml [hunk 3]
```diff
@@ -1834,20 +1834,23 @@
...
             <English>Enables weapon sway influenced by sway factors, such as stance, fatigue and medical condition.\nDisabling this setting will defer sway to vanilla or other mods.</English>
             <Japanese>姿勢、疲労、負傷状態などの手ぶれ要因に影響を受ける武器照準の揺れを有効にします。\nこの設定を無効にすると、手ぶれの揺れはバニラまたは他のMODの処理に任されます。</Japanese>
             <Korean>흔들림 계수, 자세, 피로도, 건강 상태 등의 요인에 영향을 받는 무기 흔들림을 활성화합니다.\n이 설정을 비활성화하면 바닐라 또는 다른 모드의 흔들림으로 대체됩니다.</Korean>
             <French>Active l'oscillation de l'arme influencé par les facteurs d'oscillation, tels que la position, la fatigue et l'état de santé.\nLa désactivation de ce paramètre reportera l'oscillation à vanilla ou à d'autres mods.</French>
             <Russian>Активируйте колебание оружия в зависимости от таких факторов, как стойка, усталость и состояние здоровья.\nОтключение этого параметра приведет к переносу раскачивания на vanilla или другие моды.</Russian>
+            <German>Ermöglicht die Beeinflussung des Waffen wackeln durch Beeinflussungsfaktoren wie Haltung, Müdigkeit und Gesundheitszustand.\nDurch Deaktivieren dieser Einstellung erlaubt die Beeinflussung durch Vanilla oder andere Mods.</German>
```
Comment (BaerMitUmlaut) [inline addons/common/stringtable.xml:1853]:
```suggestion
            <German>Ermöglicht die Beeinflussung des Waffen wackeln durch Beeinflussungsfaktoren wie Haltung, Müdigkeit und Gesundheitszustand.\nDie Deaktivierung dieser Einstellung erlaubt die Beeinflussung durch Vanilla oder andere Mods.</German>
```

**Review by BaerMitUmlaut — DISMISSED** [review]

File: addons/arsenal/stringtable.xml (hunk 1 above)
Comment (Nilia119) [reply addons/arsenal/stringtable.xml]: Geändert in "Thermal und in Primärwaffe integriert"

File: addons/common/stringtable.xml (hunk 2 above)
Comment (Nilia119) [reply addons/common/stringtable.xml:1837]: Waffen schwingen | Waffen Bewegung?
Waffenunruhe, finde ich zu uneindeutig/am Punkt vorbei (deshalb hatte ich das auch nicht genommen)

File: addons/overheating/stringtable.xml [hunk 4]
```diff
@@ -914,6 +918,7 @@
             <Korean>제거 불가</Korean>
             <Russian>Несъемный</Russian>
             <French>Inamovible</French>
+            <German>Nicht entfernbar</German>
```
Comment (johnb432) [inline addons/overheating/stringtable.xml:921]: I'm not sure if it's correct in this context, so take it with a grain of salt:
```suggestion
            <German>Unentfernbar</German>
```

File: addons/overheating/stringtable.xml [hunk 5]
```diff
@@ -890,6 +891,7 @@
             <Korean>오픈 볼트</Korean>
             <Russian>Открыть болт</Russian>
             <French>Obturateur ouvert</French>
+            <German>Öffne den Verschluss</German>
```
Comment (johnb432) [inline addons/overheating/stringtable.xml:894]: Definitely wrong, it's not an action, as in "opening the bolt".
We are talking about the bolt type here, so if the weapons fires from an open bolt (many SMGs and MGs do) vs closed bolt (rifles usually).

File: addons/overheating/stringtable.xml [hunk 6]
```diff
@@ -898,6 +900,7 @@
             <Korean>클로즈드 볼트</Korean>
             <Russian>Закрыть болт</Russian>
             <French>Obturateur fermé</French>
+            <German>Schließe den Verschluss</German>
```
Comment (johnb432) [inline addons/overheating/stringtable.xml:903]: Same here.

File: addons/maptools/stringtable.xml [hunk 7]
```diff
@@ -429,6 +445,7 @@
             <Italian>Su</Italian>
             <Russian>Вверх</Russian>
             <French>Monter</French>
+            <German>Hoch</German>
```
Comment (johnb432) [inline addons/maptools/stringtable.xml:448]: I think "nach oben" or just "oben" would be more appropriate here, but I only have partial context, so take it with a grain of salt.

**Review by johnb432 — COMMENTED** [review]
My German isn't the best, so take everything with a grain of salt.

File: addons/overheating/stringtable.xml (hunk 4 above)
Comment (Nilia119) [reply addons/overheating/stringtable.xml:921]: Nicht herausnehmbar [The literal translation would be you can't take the barrel out of the Gun]? In this context, you wouldn't do "un" the 1 on 1 translation would be unremovable [translation wise, it fits, but you just don't use it in this context].

File: addons/maptools/stringtable.xml (hunk 7 above)
Comment (Nilia119) [reply addons/maptools/stringtable.xml]: Makes more sense, changed it :D

**Comment by Nilia119** [comment]
> My German isn't the best, so take everything with a grain of salt.

All input is good :D. Sometimes you can't see the forest for the trees xD.

**Comment by Nilia119** [comment]
Do you mind merging this?
I was hoping, the translations would be in the next update.
I **really really really** hate having to resolve conflicts. Not only that, but I changed everything, which was requested, sucks this is lying here for three weeks now... I won't fix any more conflicts. Thank you!

File: addons/arsenal/stringtable.xml [hunk 8]
```diff
@@ -1254,7 +1255,8 @@
             <Russian>Интегрирован тепловизор и осн.прицел.</Russian>
             <Korean>열화상과 주무기 내장</Korean>
             <French>Thermique et primaire intégrés</French>
-            <Spanish>Térmica y Primaria integrada</Spanish>
+            <German>Thermal und in Primärwaffe integriert</German>
+          <Spanish>Térmica y Primaria integrada</Spanish>
```
Comment (Mike-MF) [inline addons/arsenal/stringtable.xml:1259]:
```suggestion
            <Spanish>Térmica y Primaria integrada</Spanish>
```

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**